Firmware/ultralcd.cpp

```diff
@@ -993,27 +995,36 @@ void lcd_commands()
                 was_enabled = thermal_model_enabled();
                 thermal_model_set_enabled(false);
 #endif //THERMAL_MODEL
-                lcd_commands_step = 2;
 #else //nozzle change without heating
-                enquecommand_P(G28W);
-                enquecommand_P(PSTR("G1 X125 Z200 F1000"));
-                enquecommand_P(PSTR("M109 S0"));
-                lcd_commands_step = 1;
+                if((int)degHotend(active_extruder)>40) { //check temp
```

Comment:
I think 40°C is too hot to touch.

Reply:
Body temp is 37°. 40° is safe and saves time.